libs/core/src/components/sage-tabs/sage-tabs.tsx

```diff
@@ -0,0 +1,132 @@
+import { Component, Element, Host, h, Prop, Listen } from '@stencil/core';
+import { move } from 'fs-extra';
+
+  /**
+ * @slot tabs - Content is placed within the `div[role="tablist"]` element as children
+ * @slot tabpanels - Content is placed directly after the `div[role="tablist"]` element as siblings
+ */
+@Component({
+  tag: 'sage-tabs',
+  styleUrl: 'sage-tabs.scss',
+  shadow: true,
+})
+export class SageTabs {
+  private tabs;
+  private tabPanels;
+
+  @Element() el: HTMLDivElement;
+
+  /**
+   * Sets the aria-label attached to the tablist element
+  */
+  @Prop() tablistLabel!: string;
+ 
+  /**
+    * Sets unique id on tabs component
+  */
+  @Prop() componentId!: string;
+  
+  /**
+   * Sets tabs variant styles as outlined in Figma documentation
+  */
+  @Prop() variant!: 'primary' | 'availability' | 'filter';
+
+  /**
+   * Sets starting active tab and maintains active tab as component re-renders
+   */
```

Comment:
Since this is a string, I would change the description a bit.

`Sets the starting active tab name and maintains the name as the component re-renders`